horreum-api/src/main/java/io/hyperfoil/tools/horreum/entity/json/DataSet.java

```diff
@@ -0,0 +1,65 @@
+package io.hyperfoil.tools.horreum.entity.json;
+
+import java.time.Instant;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+
+import org.hibernate.annotations.Type;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+import io.smallrye.common.constraint.NotNull;
+
+@Entity(name="dataset")
+@RegisterForReflection
+/**
+ * Purpose of this object is to represent derived run data.
+ */
+public class DataSet extends PanacheEntityBase {
+
+   public static final String EVENT_NEW = "dataset/new";
+   public static final String EVENT_TRASHED = "dataset/trashed";
+
+   @Id
+   @SequenceGenerator(
+         name="datasetSequence",
+         sequenceName="dataset_id_seq",
+         allocationSize=1)
+   @GeneratedValue(strategy=GenerationType.SEQUENCE,
+         generator= "datasetSequence")
+   public Integer id;
+
+   @NotNull
+   @Column(name="start", columnDefinition = "timestamp")
+   public Instant start;
+
+   @NotNull
+   @Column(name="stop", columnDefinition = "timestamp")
+   public Instant stop;
+
+   public String description;
+
+   @NotNull
+   public Integer testid;
+
+   @NotNull
+   @Type(type = "io.hyperfoil.tools.horreum.entity.converter.JsonUserType")
+   public JsonNode data;
+
+   @NotNull
+   @Column(columnDefinition = "boolean default false")
+   public boolean trashed;
```

Comment:
We don't need trashed datasets - a run that is trashed shouldn't map to any datasets, and when it's untrashed the datasets should be created again.

Reply:
ok I'll remove this field.